### PR TITLE
Update example repos that were moved to sclorg

### LINKED ISSUE #2803: Update example repos that were moved to sclorg
Many of the GitHub repos for the example templates (ex: django-ex) have been moved to sclorg.

These should be updated so we don't rely on the redirect.

### PR DESCRIPTION
**Pull request by liangxia**
Fixes #2803

### CODE REVIEW

**Comment by gbraad** [comment]
add to whitelist

File: test/integration/features/basic.feature [hunk 1]
```diff
@@ -139,7 +139,7 @@ Feature: Basic
    Given Minishift has state "Running"
     When executing "oc login --username=developer --password=developer" succeeds
      And executing "oc new-project ruby" succeeds
-     And executing "oc new-app centos/ruby-22-centos7~https://github.com/openshift/ruby-ex.git" succeeds
+     And executing "oc new-app centos/ruby-22-centos7~https://github.com/sclorg/ruby-ex.git" succeeds
```
Comment (praveenkumar) [inline test/integration/features/basic.feature]: :+1:

File: docs/source/getting-started/quickstart.adoc [hunk 2]
```diff
@@ -77,7 +77,7 @@ The following steps describe how to deploy a sample Node.js application from the
 .  Create a Node.js example app:
 +
 ----
-$ oc new-app https://github.com/openshift/nodejs-ex -l name=myapp
+$ oc new-app https://github.com/sclorg/nodejs-ex -l name=myapp
```
Comment (praveenkumar) [inline docs/source/getting-started/quickstart.adoc]: :+1:

File: test/integration/features/cmd-openshift.feature [hunk 3]
```diff
@@ -50,7 +50,7 @@ cluster in VM provided by Minishift.
 
   Scenario: User deploys nodejs example application from OpenShift repository
      Given Minishift has state "Running"
-      When executing "oc new-app https://github.com/openshift/nodejs-ex -l name=myapp" retrying 10 times with wait period of 2 seconds
+      When executing "oc new-app https://github.com/sclorg/nodejs-ex -l name=myapp" retrying 10 times with wait period of 2 seconds
```
Comment (praveenkumar) [inline test/integration/features/cmd-openshift.feature]: :+1:

**Comment by praveenkumar** [comment]
retest this please.

**Comment by liangxia** [comment]
280 scenarios (276 passed, 4 failed)
Failed steps:
   Scenario: Pods docker-registry and router are ready before OpenShift restart # features/cmd-openshift.feature:29
   Scenario: Pods docker-registry and router are ready after OpenShift restart # features/cmd-openshift.feature:45
   Scenario: Deployed Node.js application is available # features/cmd-openshift.feature:115
   Scenario: Provision latest OpenShift version # features/provision-various-versions.feature:34

Those 4 failed scenarios are not related to the change in this PR.

**Comment by praveenkumar** [comment]
retest this please.

**Comment by gbraad** [comment]
Thank you for your contribution.

**Pull request merged**